examples/live/create_drm_live_encoding.go

```diff
@@ -0,0 +1,521 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/bitmovin/bitmovin-go/bitmovin"
+	"github.com/bitmovin/bitmovin-go/bitmovintypes"
+	"github.com/bitmovin/bitmovin-go/models"
+	"github.com/bitmovin/bitmovin-go/services"
+)
+
+const (
+	MaxRetries int = 500
```

Comment:
This is technically correct golang, however, it is the one and only place I truly disagree with the spec, because I feel like there is a tight correlation between variables in all caps, and them being constants, I prefer there being a different between mutable and unmutable values, but golang doesn't feel that way.  I have all caps constants elsewhere in the code, but I'm okay with this.